npc/merchants/advanced_refiner.txt

```diff
@@ -93,7 +93,7 @@ payon,157,146,6	script	Suhnbi#cash	4_M_03,{
 
 	mes "[Suhnbi]";
 	mes "Clang! Clang! Clang!";
-	if (getequippercentrefinery(.@part) > rand(100) || getequippercentrefinery(.@part) > rand(100)) {
+	if (getequippercentrefinery(.@part, true) > rand(100)) {
```

Comment:
hmm ... `true`?? aren't suppose to use the documented constant here? like `REFINE_CHANCE_TYPE_XXXXX` ?